rpc/api/personal.go

```diff
@@ -98,9 +98,22 @@ func (self *personalApi) NewAccount(req *shared.Request) (interface{}, error) {
 	if err := self.codec.Decode(req.Params, &args); err != nil {
 		return nil, shared.NewDecodeParamError(err.Error())
 	}
-
+	var passwd string
+	if args.Passphrase == nil {
+		fe := self.xeth.Frontend()
+		if fe == nil {
+			return false, fmt.Errorf("No password provided")
+		}
+		var ok bool
+		passwd, ok = fe.AskPassword()
+		if !ok {
+			return false, nil
```

Comment:
Return an error with the reason that the account could not be created because the password could not be asked.